lib/Widgets/Settings.vala

```diff
@@ -36,6 +36,25 @@ namespace Granite {
             LIGHT
         }
 
+        private string? _accent_color = null;
+
+        /**
+         * The theme accent color chosen by the user
+         * @since 7.5.0
+         */
+        [Version (since = "7.5.0")]
+        public string accent_color {
```

Comment:
Any reason to do string and not Gdk.RGBA?

Reply:
Probably not. RGBA would probably be the better/more flexible choice. Good call